doc/README.md

```diff
@@ -30,6 +30,7 @@ Drag Bitcoin Core to your applications folder, and then run Bitcoin Core.
 
 * See the documentation at the [Bitcoin Wiki](https://en.bitcoin.it/wiki/Main_Page)
 for help and more information.
+* Ask for help on [the Bitcoin StackExchange](https://bitcoin.stackexchange.com)
```

Comment:
nit: to be consistent with line 31 above, `the` shouldn't be part of the link

Reply:
done